### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by habatkasa**
hello guys this is how far i have gone so far.   
please give me some feed back.
@armstrongsouljah @tomuhenry @sanya-kenneth @Masete @codjoero @mariamiah

### CODE REVIEW

File: UI/admin.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    <li><a href="#" class="menue_item">View Sales records</a></li>
+                    <li><a href="#" class="menue_item">Home</a></li>
+                    <li><a href="#" class="menue_item">New user account</a></li>
+                    <li><a href="#" class="menue_item">Edit Products</a></li>
+                    <li><a href="#" class="menue_item">View Sales records</a></li>
+                </ul>
+            </div>
+            <div class="admin_display">
+                <nav>
+                        <a href="#" class="display_item">ID</a>
```
Comment (tomuhenry) [inline UI/admin.html]: It's always a good practice to keep the indentation uniform... Use flake8 extension in VScode to help with formatting the documents properly according to pep standards
@habatkasa

**Comment by tomuhenry** [comment]
Could you merge the changes to gh-pages and share the link so that we could see how the pages appear in the browser

File: UI/admin.html (hunk 1 above)
Comment (armstrongsouljah) [reply UI/admin.html]: @tomuhenry, regarding the pep standards, that applies to python so is the flake8 module. He could instead get an html linting extension.

**Review by armstrongsouljah — COMMENTED** [review]
Hey, having looked through the code, I noticed that you are using css classes that only carry meaning you and this is going to be hard for a person trying to maintain your code later. Forexample `mostcontents` that could be replaced by something like `main-container` or `main-content`.
Then on the seperation of concerns, since we have attendants and admin, you could create `dirs` for both so that things regarding admin are put in `/admin` and the latter to `/attendant`.
I'd also like you to always link your `gh-pages` to enable for a swift review. Happy coding.

File: UI/admin.html (hunk 1 above)
Comment (tomuhenry) [reply UI/admin.html]: @armstrongsouljah ohh.. cool, didn't know that. I thought flake8 also works on other types coz it's the one I have installed and just Alt+Shift+F to format all doc types.
@habatkasa get that linting extension

**Comment by codjoero** [comment]
good comments @armstrongsouljah

File: UI/admin.html (hunk 1 above)
Comment (mariamiah) [reply UI/admin.html]: There's an extension called beautify in vscode, he could try it too..it works

Comment (habatkasa) [reply UI/admin.html]: points noted with thanks @tomuhenry  @armstrongsouljah  @mariamiah

**Pull request merged**